packages/hash/frontend/src/components/hooks/useCreatePage.ts

```diff
@@ -1,24 +1,27 @@
 import { useMutation } from "@apollo/client";
 import { useRouter } from "next/router";
 
-import { createPage } from "@hashintel/hash-shared/queries/page.queries";
+import {
+  createPage,
+  setParentPage,
+} from "@hashintel/hash-shared/queries/page.queries";
+import {
+  SetParentPageMutation,
+  SetParentPageMutationVariables,
+} from "@hashintel/hash-shared/graphql/apiTypes.gen";
```

Comment:
Placed it there because other page related queries/mutations (createPage/updatePage/getPageQuery) are placed there. I think it might be needed in collab too

Reply:
I'm not sure why collab would need to set the parent page of a page, I think queries should be in the frontend package until they really do need to be shared between collab and the frontend.

Reply:
Moved `setParentPage` query to the frontend package here => [ae953a5](https://github.com/hashintel/hash/pull/409/commits/ae953a52dbddd07fa8cd9b5c1b0728762ab48e68)

Reply:
`createPage` could also be moved out as I don't see where it is being used, but I see that wasn't introduced as part of this PR so doesn't have to be changed as part of it.

Reply:
Added a todo comment for that here => [e9342b0](https://github.com/hashintel/hash/pull/409/commits/e9342b05cc8bd16c57a815841d635ed7724bedf0)